docs/manual/src/glossary.md

```diff
@@ -0,0 +1,26 @@
+# UniFFI Glossary
+
+This page contains terms used throughout these docs
+
+## Scaffolding
+
+UniFFI-generated Rust code.
+This is an `extern "C"` API that helps the foreign bindings make Rust calls.
+
+## Bindings
+
+UniFFI-generated code in a foreign language.
+This implements the exported Rust API by calling in to the Rust scaffolding.
+
+## Lower
+
+The process of converting a high-level Rust/foreign language type into a lower-level FFI type as
```

Comment:
At first read I thought this was just from the scaffolding perspective. Maybe: "The process of converting a high-level Rust type (when used in the context of the Scaffolding) or foreign language type (when used in the context of the Bindings) ..."? Ditto for lower

(but if you think that's worse, I'm fine with your version - we can improve it later if needed)